src/serverless/functions/token.js

```diff
@@ -6,18 +6,12 @@ const VideoGrant = AccessToken.VideoGrant;
 const MAX_ALLOWED_SESSION_DURATION = 14400;
 
 module.exports.handler = async (context, event, callback) => {
-  const {
-    ACCOUNT_SID,
-    TWILIO_API_KEY_SID,
-    TWILIO_API_KEY_SECRET,
-    API_PASSCODE,
-    API_PASSCODE_EXPIRY,
-    DOMAIN_NAME,
-    ROOM_TYPE,
-  } = context;
+  const { ACCOUNT_SID, TWILIO_API_KEY_SID, TWILIO_API_KEY_SECRET, ROOM_TYPE } = context;
 
-  const { user_identity, room_name, passcode, create_room = true } = event;
-  const [, appID, serverlessID] = DOMAIN_NAME.match(/-?(\d*)-(\d+)(?:-\w+)?.twil.io$/);
+  const authHandler = require(Runtime.getAssets()['/auth-handler.js'].path);
```

Comment:
Nice 🎉

Reply:
I have a concern with this approach. See my comment in `src/serverless/functions/recordingrules.js`